lib/datasource/terraform-provider/index.ts

```diff
@@ -116,4 +124,122 @@ export class TerraformProviderDatasource extends TerraformDatasource {
     logger.trace({ dep }, 'dep');
     return dep;
   }
+
+  @cache({
+    namespace: `datasource-${TerraformProviderDatasource.id}-builds`,
+    key: (registryURL: string, repository: string, version: string) =>
+      `${registryURL}/${repository}/${version}`,
```

Comment:
can the result change? or should we cache this infos much longer? eg ttl=1w ?

Reply:
Theoretically not, but it is possible that the supplier adds build arches after the fact. As I am aware Hashicorp doesn't do this, but as we are supporting private registries users potentially can do this.

Reply:
That makes me thinking about caching of the `createHashes` function. Is there a way for the user to invalidate the cache?    
I'm talking about the RenovateApp for self hosted setups it is obvious.